staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -190,71 +192,97 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	case "get":
 		handler := handlers.GetResource(storage, storage, requestScope)
 		handler(w, req)
-		return
```

Comment:
keep the returns, it saves scanning to the end of the switch block to check behavior since the function is long

Reply:
ok, that's the intention.